src/main/host/descriptor/file.h

```diff
@@ -19,6 +19,9 @@
 #include "main/core/support/definitions.h"
 #include "main/host/syscall/kernel_types.h"
 
+/* Shadow emulates only specific flags. */
```

Comment:
I'm not quite following this comment. Is this the set of flags that we emulate in Shadow instead of just passing them through to the native file?

Reply:
Right, they're the flags the are handled within Shadow and aren't passed to the native file. Expanded the comment.